lib/commands/users.ts

```diff
@@ -75,6 +75,113 @@ export const createUser = function(user: User): Cypress.Chainable<Cypress.Respon
 		// Avoid that any follow up request reuses the admin cookies
 		cy.clearCookies()
 
+		return cy.wrap(response)
```

Comment:
Was there an issue?
I think returning is done automatically from the cy.request.
You can see it was working before on the cypress tests
https://github.com/nextcloud/nextcloud-cypress/blob/9d2a49920db654c2a3a6ad2228fd9b6cfed23319/cypress/e2e/users.cy.ts#L39-L41

Reply:
No I did not change anything here, this is just some git diff issue (see end of file), this line was previously there:
https://github.com/nextcloud/nextcloud-cypress/blob/e1c8ef9c0d940660735efbd608546f242ea75141/lib/commands/users.ts#L78